### DISCUSSION TITLE
HELP - I no longer see the commits of branch, seems all "flatten"

### DISCUSSION THREAD

**Discussion by stecydube** [gitextensions/gitextensions#10257] [Q&A] [answered]
I don't know what I did but I have only the first commit of each branch and no longer have all the previous commits in the tree view (and this also messes up the diff)

![image](https://user-images.githubusercontent.com/5737502/195840245-593b8dba-eeb4-45fc-b872-d5af640aad16.png)

Using the console I see all the commits, fortunately, so it's not git having a memory loss 😁

![image](https://user-images.githubusercontent.com/5737502/195840521-67d3704e-4570-40dd-a6a7-a88a3a805872.png)

Help !

**Comment by stecydube** [comment, marked as answer]
Geez, I searched the issues and didn't find anything but going through google search provided me the answer.

View->Set advanced filter
Uncheck _Simplify by Decoration_

At first, I saw that it was checked but I could not uncheck it until I temporarily checked _Branches_ and now I could uncheck the other.

Phew

**Comment by gerhardol** [comment]
The dependency is decoupled in 4.0 beta
Also, active filters are indicated on the filter button and there is a tooltip

![image](https://user-images.githubusercontent.com/6248932/195952461-7ac77979-acf5-4321-891e-645ece176749.png)
